resources/docker/web/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM fpfis/httpd-php
```

Comment:
We need to change this to `FROM fpfis/httpd-php-dev:7.1`. I tried on a colleague laptop and the wrong image was pulled.